webapp/src/monaco.tsx

```diff
@@ -69,8 +69,10 @@ class CompletionProvider implements monaco.languages.CompletionItemProvider {
         const wordEndOffset = model.getOffsetAt({ lineNumber: position.lineNumber, column: word.endColumn })
 
 
+        pxt.perf.measureStart("getCompletions")
```

Comment:
>pxt.perf.measureStart("getCompletions") [](start = 8, length = 39)

I think you can already get this measurement by setting `pxt.appTarget.compile.switches.time`

Reply:
Hmm, that's not giving me anything?
![image](https://user-images.githubusercontent.com/6453828/88345196-51885180-ccfa-11ea-9c84-4a9c99eb1219.png)

Reply:
Ah I get it now:
![image](https://user-images.githubusercontent.com/6453828/88345281-7d0b3c00-ccfa-11ea-95e4-caf9949528f9.png)

Thanks! didn't know this

Reply:
I found it while tracing through the code. I suspect it might be a config setting read from a file.

---
In reply to: [459766521](https://github.com/microsoft/pxt/pull/7351#discussion_r459766521) [](ancestors = 459766521)